pontoon/base/templates/header.html

```diff
@@ -23,6 +23,12 @@
       <div class="right">
         {{ Notifications.menu() }}
 
+        <div class="sign-in-header">
+          {% if not user.is_authenticated() %}
```

Comment:
I believe you should move this line a line up and wrap the entire `.sign-in-header` block. Otherwise part of the markup stays in regardless of the and the CSS adds unwanted padding.

Reply:
Sorry, I didn't get you. Which line are you talking about? I removed the extra padding to the notifications icon.

Reply:
It should look like this:
```jinja2
{% if not user.is_authenticated() %}
<div class="sign-in-header">
    <a id="fxa-sign-in" href="{{ provider_login_url(request) }}" class="button">Sign Up</a>
</div>
{% endif %}
```